### PR TITLE
Browser page styling & disconnect button

### PR DESCRIPTION
**Pull request by samepant**
Screen Shot 2021-12-02 at 7 31 26 PM
Screen Shot 2021-12-02 at 7 31 19 PM

### CODE REVIEW

File: src/components/Box/style.module.css [hunk 1]
```diff
@@ -6,8 +6,17 @@
   background-color: rgba(217, 212, 173, 0.1);
   background-clip: padding-box;
 }
-.double {
-  border: 4px double rgba(217, 212, 173, 0.3);
+
+.double:before {
```
Comment (jfschwarz) [inline src/components/Box/style.module.css:10]: What's the reason for using this more complex pseudo element approach over simply `border: 4px double rgba(217, 212, 173, 0.3);`? 

Currently the background color reaches into the space between the borders. In the original design it doesn't.
image vs. image

**Comment by jfschwarz** [comment]
I love the address stack 😍

File: src/browser/index.module.css [hunk 2]
```diff
@@ -18,4 +18,42 @@
 
 .frame {
   height: 100%;
+  max-width: 1200px;
```
Comment (jfschwarz) [inline src/browser/index.module.css:21]: I'd say we shouldn't set a max width. Depending on the Dapp UI users may want to leverage to full width of their screen. It also breaks the alignment of the browser frame with the address bar.

File: src/components/Box/style.module.css (hunk 1 above)
Comment (samepant) [reply src/components/Box/style.module.css:10]: The psuedo element approach allows for a little more flexibility and exactness, I know they look pretty similar but the pseudo approach is a little more specific. 

The background reaching between the lines is intentional (with the pseudo approach supports no background between the border as well). I like the filled double border a little better, it brings more focus and specificity to the interactive boxes.

 The figma designs are an initial sketch, but I like to continue tweaking the look and feel while ironing out the css, I always felt like in the code was a more powerful place to do final design work

File: src/browser/index.module.css (hunk 2 above)
Comment (samepant) [reply src/browser/index.module.css:21]: I'm good with removing the max-width until we have the sidebar. i was working on a big monitor and the Uniswap iframe felt too massive

**Comment by samepant** [comment]
> I love the address stack 😍
> 
> To me the following order of addresses in the stack would make most sense:
> 
> ```
> WalletConnected Address > Target Module Address > Avatar Address
> ```
> 
> Like this, each address controls/calls to the address to their right
> 
> **edit:**
> 
> Actually the other way around would also work: each module plugs into the module to their left.
> 
> ```
> Avatar Address <  Target Module Address < WalletConnected Address
> ```
 Ah ok this makes sense. I was confused about which addresses were which. The avatar address is the safe and the target address is the zodiac module?

**Comment by jfschwarz** [comment]
> The avatar address is the safe and the target address is the zodiac module?

Correct. We tried following the names from the zodiac contracts: 

- Avatar: The Safe being controlled, holding the treasury
- Target: The address a module–or in this case the Pilot app–calls to.

**Pull request merged**